ENIGMAsystem/SHELL/Platforms/Win32/WINDOWScallback.cpp

```diff
@@ -203,12 +202,12 @@ namespace enigma
          hdeltadelta %= WHEEL_DELTA;
          return 0;
 
-      case WM_LBUTTONUP:   { if (!wParam) ReleaseCapture(); mousestatus[0]=0; return 0; }
-      case WM_LBUTTONDOWN: SetCapture(hWnd); mousestatus[0]=1; return 0;
-      case WM_RBUTTONUP:   { if (!wParam) ReleaseCapture(); mousestatus[1]=0; return 0; }
-      case WM_RBUTTONDOWN: SetCapture(hWnd); mousestatus[1]=1; return 0;
-      case WM_MBUTTONUP:   { if (!wParam) ReleaseCapture(); mousestatus[2]=0; return 0; }
-      case WM_MBUTTONDOWN: SetCapture(hWnd); mousestatus[2]=1; return 0;
+      case WM_LBUTTONUP:   { if (!wParam) ReleaseCapture(); input_mouse_up(0); return 0; }
```

Comment:
This isn't a 1:1 replacement. Where is the deletion for the logic that set the previous mouse state, before?

Reply:
We were never doing that on Windows or else it was deleted a long time ago....